spec/system/candidate_interface/course_selection/candidate_edits_course_choices_with_apply_again_with_three_choices_deactivated_spec.rb

```diff
@@ -0,0 +1,282 @@
+require 'rails_helper'
+
+RSpec.feature 'Candidate edits course choices' do
+  include CandidateHelper
+  include CourseOptionHelpers
+
+  scenario 'Candidate is signed in' do
+    given_i_am_signed_in
+    and_the_apply_again_with_three_choices_feature_flag_is_deactivated
```

Comment:
It feels a bit odd that we're moving the tests _and_ changing the feature flag— why not add this call to the existing spec?

Reply:
This is to optimize the removal of the feature flag in the future. When the feature flag is active and it is working as expected, when we need to remove the code, the specs can be removed easily because they were renamed.